src/ae_apidata.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * ae_apidata.h - Header file for API data functions implemented in ae.c
+ *
+ * This header file declares functions that are implemented in ae.c.
+ * These functions may alter the state of the underlying apidata    ,
+ * which may require a mutex for modification in the case of the server
+ * main event loop (`server.el`), as it may be accessed by IO threads.
+ */
+
+#ifndef AE_APIDATA_H
+#define AE_APIDATA_H
+
+#include "ae_eventloop.h"
+
+int aeCreateFileEvent(aeEventLoop *eventLoop, int fd, int mask, aeFileProc *proc, void *clientData);
+void aeDeleteFileEvent(aeEventLoop *eventLoop, int fd, int mask);
+int aeResizeSetSize(aeEventLoop *eventLoop, int setsize);
+int aePoll(aeEventLoop *eventLoop, struct timeval *tvp);
```

Comment:
Now when I see this implementation, I realize that `#include "ae_apidata.h"` and `#include "ae.h"` are equivalent. Nobody needs to use `#include "ae_apidata.h"`.

Thus, we can actually put these 4 prototypes directly in `ae.h` and we don't need `ae_apidata.h`. I will be even simpler, right?

Reply:
Oh right, thanks, updated.